### DISCUSSION TITLE
How to modify and test .fx effects files without restarting MSFS？

### DISCUSSION THREAD

**Discussion by KyleCyaN** [flybywiresim/aircraft#7144] [Help]
Hi, I have a issue that I love tweaking these .fx effects files to my standards, but each modification requires a restart of the game for the new effects to take effect, so is there any way to modify and test the fx effects file without restarting the game?

![image](https://user-images.githubusercontent.com/66722059/166161599-42157d66-9fc1-4498-a243-abf9b53dd006.png)

**Comment by aguther** [comment]
Maybe better asked on https://devsupport.flightsimulator.com
